### DISCUSSION TITLE
LoopDocs

### DISCUSSION THREAD

**Discussion by eStarkmann** [community/community#70135] [Programming Help]
### Body

Hi
I want to install a Loop from Loop Docs, in starting task "Build Loop" I am getting to following error with building Loop with Browser: built/ archive 
can anyone tell me what's wrong?

**Comment by gaganachiranjitha99** [comment]
test2

**Reply by KoshikaDul1999** [reply]
njh

**Reply by eStarkmann** [reply]
do you know what to do?
